awxkit/awxkit/cli/resource.py

```diff
@@ -135,7 +136,13 @@ def handle(self, client, parser):
             parser.print_help()
             raise SystemExit()
 
-        data = json.load(sys.stdin)
+        format = getattr(client.args, 'conf.format')
```

Comment:
Is there a reason you didn't just do `client.get_config('format')` ?

Reply:
I simply wasn't aware of it :smile: